docs/tutorial/using-checkbox/test-report.rst

```diff
@@ -133,8 +133,10 @@ in the search bar, you can view all test results related to audio testing.
     The xz compressed tarball is a comprehensive archive that includes the 
     aforementioned reports and all associated attachments, such as I/O logs 
     and binary files. You can extract the tarball with ``tar -xf 
-    sumbission.tar.xz -C /path/to/destination``.
+    submission.tar.xz -C /path/to/destination``.
 
     Certification Website only accepts submissions tarballs, from which it 
     extracts the ``submission.json`` file to create a new test report in the 
-    database. 
\ No newline at end of file
+    database. 
+
+For a detailed description of the ``submission.json`` file, see the :doc:`../../reference/submission-schema` reference.
```

Comment:
There's a reference made to "submission files" also right at the start of the document. Links to  the reference would be helpful from there too. Also here, IMO it would be clearer if what you mention about the JSON file was made under the "JSON files store session exports ..." already. IMO now that we document the file format, it would be clearer to not refer to it as a thing that relates to the certification website, since it can be used for... whomever for whatever much better now since it is documented (will also become a stable a definition for the schema from the point we start validating against it).

You mentioned on mattermost that this document was not a part of the tutorial, was that in fact resolved?

Reply:
The clean up work is to be followed up with https://warthogs.atlassian.net/browse/CHECKBOX-1069